### PR TITLE
stm32 nucleo-g0b1re fixes

### PR DESCRIPTION
**Pull request by raul-klg**
Hello. I'd like to help with some little changes/fixes to the recently added support to stm32g0b1 and nucleo-g0b1re, specifically to the related dts files.

### CODE REVIEW

**Review by erwango — CHANGES_REQUESTED** [review]
Thanks for these fixes!
Aside from the compliance checks, looks good.
Please fix the reported long lines in commit messages (which otherwise are well documented, thanks for that also).

File: boards/arm/nucleo_g0b1re/arduino_r3_connector.dtsi [hunk 1]
```diff
@@ -16,8 +16,8 @@
 			   <3 0 &gpiob 1 0>,	/* A3 */
 			   <4 0 &gpiob 11 0>,	/* A4 */
 			   <5 0 &gpiob 12 0>,	/* A5 */
-			   <6 0 &gpioa 3 0>,	/* D0 */
-			   <7 0 &gpioa 2 0>,	/* D1 */
+			   <6 0 &gpioc 5 0>,	/* D0 */
+			   <7 0 &gpioc 4 0>,	/* D1 */
```
Comment (str4t0m) [inline boards/arm/nucleo_g0b1re/arduino_r3_connector.dtsi:20]: The nucleo_g071rb, uses the same board, could you eventually fix it there as well?

**Review by str4t0m — CHANGES_REQUESTED** [review]
One nit.

**Comment by jenmwms** [comment]
> Please fix the reported long lines in commit messages

To answer your question from dev mail list, there are a few ways to satisfy these compliance checks. Here, IMO you can just cut the long line onto another line to satisfy the limit. Line break between the warnings can help readability then. Or, you can file a quick bug and link the number in your commit with a Fixes #xxxxx.  In some cases, when not simple to shorten, it may be permissible based on feedback from approvers.

**Review by erwango — APPROVED** [review]

**Review by FRASTM — APPROVED** [review]

**Pull request merged**